apps/ech.c

```diff
@@ -45,11 +45,14 @@ const OPTIONS ech_options[] = {
     {"help", OPT_HELP, '-', "Display this summary"},
     {"verbose", OPT_VERBOSE, '-', "Provide additional output"},
     OPT_SECTION("Key generation"),
-    {"pemout", OPT_PEMOUT, '>', "Private key and ECHConfig [echconfig.pem]"},
+    {"pemout", OPT_PEMOUT, '>',
+     "Private key and ECHConfig [default echconfig.pem]"},
     {"public_name", OPT_PUBLICNAME, 's', "public_name value"},
-    {"mlen", OPT_MAXNAMELENGTH, 'n', "Maximum name length value"},
+    {"mlen", OPT_MAXNAMELENGTH, 'n',
```

Comment:
What about my suggestion to use `max_len` or `max_name_len` for the option? If you do not like it I do not insist, but have you considered it?

Reply:
I've no problem going with `max_name_len` so will do that. (I don't think this will see much use in any case - it assume the ECH decryptor knows the longest name that'll be used in an inner CH SNI, which seems pretty unlikely to me - it's a not-great protocol feature IMO but is what it is;-)